.github/workflows/test-template.yml

```diff
@@ -138,7 +138,7 @@ jobs:
           - '--leaf'
           - '--no-leaf'
     needs: cache-toolbox
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Updating the container makes sense, but changing the `runs-on` will just lock us in to a version instead of moving ahead when GH updates the `-latest` tag.